tests/pathfinding/test_capacity_updates.py

```diff
@@ -34,26 +33,26 @@
 
 
 def get_updatepfs_message(
-        chain_identifier: ChainID = 1,
-        channel_identifier: ChannelID = 0,
+        updating_participant: Address,
+        other_participant: Address,
+        chain_identifier=ChainID(1),
+        channel_identifier=ChannelID(0),
         token_network_address: TokenNetworkAddress = DEFAULT_TOKEN_NETWORK_ADDRESS_BYTES,
-        updating_participant: Address = None,
-        other_participant: Address = None,
-        updating_nonce: Nonce = 1,
-        other_nonce: Nonce = 0,
-        updating_capacity: TokenAmount = 90,
-        other_capacity: TokenAmount = 110,
+        updating_nonce=Nonce(1),
+        other_nonce=Nonce(0),
+        updating_capacity=TokenAmount(90),
+        other_capacity=TokenAmount(110),
         reveal_timeout: int = 2,
-        signature: Signature = '',
+        signature=Signature(bytes()),
 ) -> UpdatePFS:
     return UpdatePFS(
         canonical_identifier=CanonicalIdentifier(
             chain_identifier=chain_identifier,
             channel_identifier=channel_identifier,
             token_network_address=token_network_address,
         ),
-        updating_participant=updating_participant,
-        other_participant=other_participant,
+        updating_participant=to_checksum_address(updating_participant),
```

Comment:
Shouldn't this be `to_canonical_identifier` because the `UpdatePFS` message takes a `bytes` based `Address`  representation?

Reply:
I was confused due to #203 not being merged, yet. I don't see where `to_canonical_identifier` should come from. I'll just use `decode_hex` for now.